src/components/common/RadioGroup/RadioGroup.tsx

```diff
@@ -73,7 +73,7 @@ function Radio({ value, children, className = '', showTippy = false, tippyConten
                         }
                         onChange(e)
                     }}
-                    disabled={disabled}
+                    disabled={isDisabled || disabled}
```

Comment:
can't we use the disabled from context?

Reply:
No, because it's disabling all tabs and in this case, we just want to disable a single tab conditionally.